mesonpy/__init__.py

```diff
@@ -988,6 +988,8 @@ def _get_meson_command(
     # directly. For packages that vendor a forked Meson, the `meson.py` in the
     # root of the Meson repo can be used this way.
     if meson.endswith('.py'):
+        if not os.path.exists(meson):
+            raise ConfigError(f"Could not find specified meson: {meson}")
```

Comment:
`Could not find the specified meson: "{meson}"` maybe, adding the article and quotes around the path. I think having the quotes helps when there is white space in the filename.